docker/scripts/hardware/utils.py

```diff
@@ -100,6 +100,8 @@ def get_smart_devices():
     for file in os.listdir("/dev"):
         if re.compile("^sd[a-z]$").match(file):
             disks.append(os.path.join("/dev", file))
+        if re.compile("^nvme[0-9]$").match(file):
+            disks.append(os.path.join("/dev", file))
```

Comment:
While I don't see anything "wrong" with it per se, note that with the change below to remove the call to this function from `get_smart_diskprop()` I don't think this part would have any effect?

Reply:
I suspect this is the case as well, a remnant of some testing, removed